frontend/src/pages/privacy

```diff
@@ -0,0 +1 @@
+../../../node_modules/legal-docs/firefox_testpilot_PrivacyNotice
```

Comment:
We can also have .md files in the `frontend/src/pages` directory (or subdirectories thereof) and they will be generated like compiled-templates used to do it.